QUEST_LV_0100.tsv

```diff
@@ -13096,13 +13096,13 @@ QUEST_LV_0100_20151224_013092	Yane, stop being stubborn and take care of Melchio
 QUEST_LV_0100_20151224_013093	We're lagging behind.{nl}Let's hurry and head to the Odagingas Vacant Lot!{nl}
 QUEST_LV_0100_20151224_013094	It's over!{nl}
 QUEST_LV_0100_20151224_013095	What happened to Delmore Rephaim?{nl}What was all that light and the ground shaking...?{nl}
-QUEST_LV_0100_20151224_013096	Is that true...? So he had the end he deserved.{nl}I just don't know if this can appease all the Delmore Fortress {nl}residents that are now with the goddess.
+QUEST_LV_0100_20151224_013096	Is that true...? So he had the end he deserved.{nl}I just don't know if this can appease all the Delmore Castle {nl}residents that are now with the goddess.
 QUEST_LV_0100_20151224_013097	Delmore Rephaim is gone but... I'm still worried about Melchioras.{nl}Shall we go back now?
 QUEST_LV_0100_20151224_013098	What... what happened to Delmore Rephaim?{nl}
 QUEST_LV_0100_20151224_013099	I see...{nl}I guess it went as expected from him then...{nl}
 QUEST_LV_0100_20151224_013100	Right before he escaped he seemed almost addicted to forbidden knowledge, you see.{nl}As if there was nothing that could stop him...{nl}
 QUEST_LV_0100_20151224_013101	And I followed him, knowing fully well he was an ally of the demons.{nl}But there was no match for his character, his thirst for knowledge.{nl}
-QUEST_LV_0100_20151224_013102	When I found out he was going to use the people of Delmore Fortress to make the Kruvina...{nl}Yes, had I been able to stop him then, everyone would have been saved...{nl}
+QUEST_LV_0100_20151224_013102	When I found out he was going to use the people of the Delmore Castle to make the Kruvina...{nl}Yes, had I been able to stop him then, everyone would have been saved...{nl}
```

Comment:
When I found out he was going to use the people of the Delmore Castle to make the Kruvina...{nl}Well,